src/cljs/athens/events.cljs

```diff
@@ -105,20 +112,20 @@
 ;; TODO: change right sidebar items from map to datascript
 (reg-event-fx
   :right-sidebar/open-item
-  (fn-traced [{:keys [db]} [_ uid]]
-             (let [block     (d/pull @db/dsdb '[:node/title :block/string] [:block/uid uid])
-                   new-item  (merge block {:open true :index -1})
-                   new-items (assoc (:right-sidebar/items db) uid new-item)
-                   inc-items (reduce-kv (fn [m k v] (assoc m k (update v :index inc)))
-                                        {}
-                                        new-items)
-                   sorted-items (into (sorted-map-by (fn [k1 k2]
-                                                       (compare
-                                                         [(get-in new-items [k1 :index]) k2]
-                                                         [(get-in new-items [k2 :index]) k1]))) inc-items)]
-               (cond-> {:db (assoc db :right-sidebar/items sorted-items)}
-                 (not (:right-sidebar/open db))
-                 (assoc :dispatch [:right-sidebar/toggle])))))
+  (fn [{:keys [db]} [_ uid]]
+    (let [block     (d/pull @db/dsdb '[:node/title :block/string] [:block/uid uid])
+          new-item  (merge block {:open true :index -1})
+          new-items (assoc (:right-sidebar/items db) uid new-item)
+          inc-items (reduce-kv (fn [m k v] (assoc m k (update v :index inc)))
+                               {}
+                               new-items)
+          sorted-items (into (sorted-map-by (fn [k1 k2]
+                                              (compare
+                                                [(get-in new-items [k1 :index]) k2]
+                                                [(get-in new-items [k2 :index]) k1]))) inc-items)]
+      (cond-> {:db (assoc db :right-sidebar/items sorted-items)}
+              (not (:right-sidebar/open db))
+              (assoc :dispatch [:right-sidebar/toggle])))))
```

Comment:
Need to remove `fn-traced` and fix indentation. 10x was giving a weird error on web.